### DISCUSSION TITLE
I cannot verify my Github Student Profile for Turkey Schools.

### DISCUSSION THREAD

**Discussion by ismailcaakir** [community/community#54845] [GitHub Education]
### Hi everyone,

Greetings,

I live in Turkey and I am a student of open education. My student certificate I got from turkiye.gov.tr is rejected by github when I apply for github education.

What should I do in this scenario.

**Comment by ryanlua** [comment]
Hi @ismailcaakir,

When you application to Global Campus is denied, they should provide a list of reasons on why it was denied. Please provide the message given to you for rejection.

I also encourage you to read this document on [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved) which might provide you additional insight on why you could have been rejected.

For now, have you provided an English translation of the certificate along with the original one and does the information you provided GitHub align with your account? As mentioned above, if you could post the rejection reason it would help with understanding how you can correct your application so you can get accepted.
